tests/robot-tests/tests/libs/common.robot

```diff
@@ -342,6 +345,11 @@ user clicks link
     ${element}=  get child element  ${parent}  link:${text}
     user clicks element  ${element}
 
+user clicks link when available
+    [Arguments]   ${text}  ${parent}=css:body
```

Comment:
This keyword is taking a `${parent}` argument that is unused.

Reply:
Will remove keyword altogether!

Reply:
Not sure if this PR is done, but noticed that this keyword is still in `common.robot` - just commenting in case its been missed

Reply:
Nope tis done now in the latest load of changes.